### PR TITLE
Added pixel_art to favicons configuration.

### PR DESCRIPTION
**Pull request by jameswilddev**
Please fill in this template.

- [-] Use a meaningful title for the pull request. Include the name of the package modified.
- [-] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.) *there are existing tests and they do not cover configuration*
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).  *could not get this to work before making a change*

Select one of these and delete the others:

If changing an existing definition:
- [-] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/itgalaxy/favicons/commit/fea9dd5091ac96164c7b61336f9292d9093ea701
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by mohsen1 — APPROVED** [review]

File: types/favicons/index.d.ts [hunk 1]
```diff
@@ -38,6 +38,8 @@ declare namespace favicons {
         version: string;
         /** Print logs to console? @default false */
         logging: boolean;
+        /** Use nearest neighbor resampling to preserve hard edges on pixel art @default false */
+        pixel_art: boolean;
```
Comment (amcasey) [inline types/favicons/index.d.ts]: Should this be optional?  It seems like it might break existing interface implementers.

**Review by amcasey — CHANGES_REQUESTED** [review]
Quick question.

File: types/favicons/index.d.ts (hunk 1 above)
Comment (mohsen1) [reply types/favicons/index.d.ts]: See https://github.com/DefinitelyTyped/DefinitelyTyped/blob/1b75aeb427c14f093eba6da775df4551fe0e1efa/types/favicons/index.d.ts#L92

Comment (amcasey) [reply types/favicons/index.d.ts]: Great.  Thanks!

**Review by amcasey — APPROVED** [review]

**Pull request merged**